docs/blog/oras-0.14-and-future.md

```diff
@@ -0,0 +1,107 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_[Feynman Zhou](https://twitter.com/FeynmanZhou), CNCF Ambassador, ACR Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the ORAS CLI recently. ORAS v0.14 introduces four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
+
+## What's new in ORAS 0.14
+
+<figure>
+  <img src="/blog/oras-0.14-and-future/carbon.svg" />
```

Comment:
The image doesn't seem to show in preview mode
I've also found .svg files dont' render consistently across devices. Which is a shame as the vector format would represent the best clarity. So, I've had to resort to png, with a white background to support dark mode views.

Reply:
Let me convert it into png